ui/src/notebooks/components/AddButtons.tsx

```diff
@@ -3,41 +3,92 @@ import React, {FC, useContext} from 'react'
 
 // Components
 import {Button, ComponentColor} from '@influxdata/clockface'
+import CellFamily from 'src/notebooks/components/CellFamily'
 
 // Constants
 import {NotebookContext} from 'src/notebooks/context/notebook.current'
 import {ResultsContext} from 'src/notebooks/context/results'
 import {PIPE_DEFINITIONS} from 'src/notebooks'
 
+// Utils
 import {event} from 'src/cloud/utils/reporting'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
+import {TypeRegistration} from 'src/notebooks'
+
+// Styles
+import 'src/notebooks/components/AddButtons.scss'
+
 interface Props {
   index?: number
   onInsert?: () => void
   eventName: string
 }
 
+const SUPPORTED_FAMILIES = [
+  {
+    name: 'Input',
+    family: 'inputs',
```

Comment:
i wish this was more tightly coupled with the TypeRegistration definition, but i dont know how

Reply:
One idea I have is to use enums since those have almost identical keys & values -- use the value for internal logic and the key for display purposes